junit-test/shared/src/test/scala/org/scalajs/junit/utils/JUnitTest.scala

```diff
@@ -137,12 +241,22 @@ abstract class JUnitTest {
     def successEvent: Output = Event("Success")
     def failureEvent: Output = Event("Failure")
 
-    def infoIfOrElseDebug(p: Boolean, msg: String): Output =
+    def formattedTestClass: String = formatClass(suiteUnderTestName, yellow)
+
+    private def exceptionClassInfo(show: Boolean, fullName: String) = {
+      if (show) formatClass(fullName, red) + ": "
+      else ""
+    }
+
+    private def formatClass(fullName: String, color: String => String) = {
+      val (pack, cls) = fullName.splitAt(fullName.lastIndexOf('.'))
+      pack + '.' + color(cls.tail)
```

Comment:
If you split right after the dot rather than right before it, it's a bit easier:
```scala
val (packAndDot, cls) = fullName.splitAt(fullName.lastIndexOf('.') + 1)
packAndDot + color(cls)
```